### ISSUE TITLE
plot not working

### ISSUE THREAD

**Issue by kikiwong333** [friosavila/stpackages#9]
![1713663650353](https://github.com/friosavila/stpackages/assets/109087563/476c38a8-e7a7-4f82-9354-c61fc68d4b6e)

**Comment by kikiwong333** [comment]
hello 
The results are inconsistent when I use commond csdid and csdid2. What is the specific reason for this?
Which result should I trust？
Thank you so much！
And run the following code：
csdid2 lnwwe  $firmlist $industriallist  $citylist $policy ,i(groupid) t(year) g(gvar)  method(drimp)  cluster( double_cluster)  notyet
csdid2 event, estat     window(-4,5) estore(cs) 
Next I run code ：
	event_plot  cs, ///
    stub_lag( Tp#) stub_lead( Tm#) plottype(scatter) ciplottype(rcap) ///
    together perturb(-0.325(0.13)0.455) noautolegend ///
    graph_opt(title(" industrial wastewater  estimator", size(medlarge)) ///
        xtitle("Periods since the event") ytitle("Average causal effect") xlabel(-4(1)5) ylabel(-0.2(1)0.2) ///
        legend(order(1 "TWFE" 2  "" 3 "Sun and Abraham (2021)" 4 "" 5 "Callaway and Sant’Anna 2021" 6 ""  ) rows(3) region(style(none))) ///
        xline(0, lcolor(gs8) lpattern(dash)) yline(0, lcolor(gs8)) graphregion(color(white)) bgcolor(white) ylabel(, angle(horizontal)) ///
    )  lag_opt1(msymbol(+) color(cranberry)) lag_ci_opt1(color(cranberry))   

The error：
No estimates found for the model "cs1". Make sure you have specified stub_lag and stub_lead correctly.	   
![1713670113415](https://github.com/friosavila/stpackages/assets/109087563/8fb7ef11-f182-488b-b3c1-95c2c21d7fcb)

What should I do？
Thank you so much！
Qi

**Comment by friosavila** [comment]
I need more information
First. Please make sure you have latest iteration of the packages
Second how many controls do you have. Do you check for collinearity with
the gvar?
Can you tabulate year gvar?
Thank you

**Comment by kikiwong333** [comment]
Thank you 
I solved it.
but I have another question: Why are the results of csdid and csdid2 different? What is the reason?
The first picture from csdid
![1713738822669](https://github.com/friosavila/stpackages/assets/109087563/dbd2c5b5-c47a-408c-a4b4-912275324a4c)
The second picture from csdid2
![1713738901194(1)](https://github.com/friosavila/stpackages/assets/109087563/eb089299-218f-48a1-abb8-89dfb5bdd711)
And I found that picture 2(csdid2) does not have "tm1" but tm8. The picture 2 has "tm1" but no "tm8"

Thank you so much
Qi
